samples/getPublicAccessPrevention.js

```diff
@@ -0,0 +1,50 @@
+// Copyright 2021 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/**
+ * This application demonstrates how to perform basic operations on buckets with
+ * the Google Cloud Storage API.
+ *
+ * For more information, see the README.md under /storage and the documentation
+ * at https://cloud.google.com/storage/docs.
+ */
+
+function main(bucketName = 'my-bucket') {
+  // [START storage_get_public_access_prevention]
+  /**
+   * TODO(developer): Uncomment the following lines before running the sample.
+   */
+  // The name of your GCS bucket
+  // const bucketName = 'Name of a bucket, e.g. my-bucket';
+
+  // Imports the Google Cloud client library
+  const {Storage} = require('@google-cloud/storage');
+
+  // Creates a client
+  const storage = new Storage();
+
+  async function getPublicAccessPrevention() {
+    // Gets Bucket Metadata and prints publicAccessPrevention value (either 'unspecified' or 'enforced').
+    const [metadata] = await storage.bucket(bucketName).getMetadata();
+    console.log(
+      `Public access prevention is ${metadata.iamConfiguration.publicAccessPrevention} for ${bucketName}.`
+    );
+  }
+
+  getPublicAccessPrevention().catch(console.error);
```

Comment:
Let's drop the `.catch(console.error)` portion of this one.  In prod you wouldn't catch the error like this, and would want to let it throw. (here and below)